Dockerfile

```
@@ -1,5 +1,6 @@
 FROM scratch
 COPY script/ca-certificates.crt /etc/ssl/certs/
 COPY dist/traefik /
+RUN mdir -p /tmp
```

`mdir` is not a valid command.

```suggestion
 WORKDIR /tmp
 WORKDIR /
```

---

lol that was supposed to be `mkdir`